### DISCUSSION TITLE
How do we respond to "Dynamic Island" on the Web?

### DISCUSSION THREAD

**Discussion by jwook2** [community/community#39534] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Um... I'm working as a web developer
I have a question.
I don't even know if it's right to write it down here.

I'm a web developer.
There is a hybrid app that I am working on.
However, the recent release of the iPhone 14 Pro caused problems.
Because of the iPhone 14 Pro's "Dynamic Island,"
UI is misaligned in Web view.
So I thought of two things.

1.Javascript detects "Dynamic Island" and handles exceptions.
2. Detect device model name in javascript and handle exceptions based on model name.

How can I make it possible?
I need a hand.

**Comment by changchangge** [comment]
I also encountered this problem.
I have a H5 page with SSR and I want to detect that if this user's iphone has Dynamic Island in node.I find that the user agent do not have message to detect it.
How can i make it possible?

**Comment by Trinhan87** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> Um... I'm working as a web developer
> I have a question.
> I don't even know if it's right to write it down here.
> 
> I'm a web developer.
> There is a hybrid app that I am working on.
> However, the recent release of the iPhone 14 Pro caused problems.
> Because of the iPhone 14 Pro's "Dynamic Island,"
> UI is misaligned in Web view.
> So I thought of two things.
> 
> 1.Javascript detects "Dynamic Island" and handles exceptions.
> 2. Detect device model name in javascript and handle exceptions based on model name.
> 
> How can I make it possible?
> I need a hand.
